webdriver/tests/classic/maximize_window/maximize.py

```diff
@@ -19,82 +23,88 @@ def test_no_browsing_context(session, closed_frame):
     assert_success(response)
 
 
-def test_fully_exit_fullscreen(session):
-    session.window.fullscreen()
-    assert is_fullscreen(session)
+def test_response_payload(session, minimal_screen_position):
```

Comment:
```suggestion
def test_response_payload(session):
```

nit: minimal_screen_position not used